routingpy/routers/google.py

```diff
@@ -369,12 +369,8 @@ def parse_direction_json(response, alternatives):
                 duration += leg["duration"]["value"]
                 distance += leg["distance"]["value"]
                 for step in leg["steps"]:
-                    geometry.extend(
-                        [
-                            list(reversed(coords))
-                            for coords in utils.decode_polyline5(step["polyline"]["points"])
-                        ]
-                    )
+                    geometry.extend(utils.decode_polyline5(step["polyline"]["points"]))
```

Comment:
yeah good catch. we refactored the polyline stuff (a few years ago IIRC), and this must've slipped.